bin/aptos

```diff
@@ -132,14 +132,14 @@ const main = async () => {
   }
 
   // Spawn a child process to execute the binary with the provided arguments.
-  if (os === "MacOS") {
+  if (os === "Windows") {
     spawn(path, process.argv.slice(2), {
       stdio: "inherit",
+      shell: true,
     });
   } else {
     spawn(path, process.argv.slice(2), {
       stdio: "inherit",
-      shell: true,
     });
```

Comment:
lol, you could also pull the "slice" out now that i've looked at this a couple times.  But, this works fine.

Note in the future that usually MacOS and linux usually work the same and windows is the outlier